packages/openmrs/src/http.js

```diff
@@ -126,14 +151,23 @@ export function post(path, data, options = {}) {
  * @param {RequestOptions} [options={}] - An object containing query params and headers for the request
  * @returns {operation}
  */
- function _delete(path, options = {}) {
-  return  async state => {
-    const [resolvedPath, resolvedOptions] = expandReferences(state, path, options);
+function _delete(path, options = {}) {
+  return async state => {
+    const [resolvedPath, resolvedOptions] = expandReferences(
+      state,
+      path,
+      options
+    );
 
-    const response = await util.request(state, 'DELETE', resolvedPath, resolvedOptions);
+    const response = await util.request(
+      state,
+      'DELETE',
+      resolvedPath,
+      resolvedOptions
+    );
 
     return util.prepareNextState(state, response);
-  }
+  };
 }
 
-export { _delete as delete };
\ No newline at end of file
+export { _delete as delete };
```

Comment:
Just a question, Not important to change this now. But why not `export function delete()`  why do we have to keep `_delete` ? It's not used anywhere right ?

Reply:
`delete` is a keyword, so you can't declare `function delete()` without syntax errors

Reply:
Noted 👍🏽